db/models.py

```diff
@@ -1 +1,45 @@
 from django.db import models
+
+
+class Race(models.Model):
+    RACE_TYPE = (
+        ("elf", "elf"),
+        ("dwarf", "dwarf"),
+        ("human", "human"),
+        ("ork", "ork")
+    )
```

Comment:
You could use `models.TextChoices` class for this purpose. It is not mandatory, just for the purpose of your familiarization with this class

Reply:
Like this? 
`class Race(models.Model):
    class RaceType(models.TextChoices):
        ELF = "elf", "elf"
        DWARF = "dwarf", "dwarf"
        HUMAN = "human", "human"
        ORK = "ork", "ork"

    name = models.CharField(
        max_length=255, unique=True, choices=RaceType.choices
    )
    description = models.TextField(blank=True)`

Reply:
Like this?
```
class Race(models.Model):
    class RaceType(models.TextChoices):
        ELF = "elf", "elf"
        DWARF = "dwarf", "dwarf"
        HUMAN = "human", "human"
        ORK = "ork", "ork"

    name = models.CharField(
        max_length=255, unique=True, choices=RaceType.choices
    )
    description = models.TextField(blank=True)
    ```

Reply:
Or like this?
```
class Race(models.Model):
    class RaceType(models.TextChoices):
        ELF = "elf"
        DWARF = "dwarf"
        HUMAN = "human"
        ORK = "ork"

    name = models.CharField(
        max_length=255, unique=True, choices=RaceType.choices
    )
    description = models.TextField(blank=True)